layouts/partials/list.html

```diff
@@ -0,0 +1,150 @@
+{{ $sideMargin  := "50px" }}
+{{ $verticalMargin := "30px" }}
+{{ $fontSize:= "12pt" }}
+
+{{ $currentPage := . }}
+{{ $curWeight := -1 }}
+
+{{ range .Site.Menus.docs }}  
+  {{ if .HasChildren }}
+    {{ $extractParent := $currentPage.Parent.Page }}
+    {{ $currentParent := (replaceRE "Page\\(\\/(.*)\\)" "$1" $extractParent) }}
+    {{ range .Children }}
+      {{ if eq $currentParent (.Parent) }}              
+        {{ if .Weight }}
+          <!-- Add content to be modified by JavaScript. -->
+          {{ if $currentPage.IsMenuCurrent "docs" . }}
+            <!-- Add filler html and use JavaScript to put in correct links. -->
+            <div style="margin: {{ $verticalMargin }} 0px">            
+              <a id="previousLink" href="javascript:void(0)" style="font-size: {{ $fontSize }}">Previous - Page</a>

```

Comment:
Changed href to "javascript:void(0)" as if an error occurs, worst case scenario are un-working links with generic filler text. outputting:
` Previous - Page         Next - Page `

To Expand:
Per testing links changed consistently for me. I made a support repo with a working example - [Pull #1395 working example](https://github.com/jhauga/documentation-sequential-links). You'll have to use "Codesapces" then localhost it. I used `php -S localhost:8000`, GitHub prompts with link, and a working (_mostly_) example. 

With the "Codespaces" localhost the links in section "API Entities" go to a table of contents like page, but this happens when clicking on side nav and clicking on sequential nav.